rest/authorization-server/src/main/resources/application.yml

```diff
@@ -10,20 +10,34 @@
 server:
   port: 8090
 
+couchdb:
+  url: http://sw360couchdb:5984
+  database: sw360oauthclients
```

Comment:
Why is this configurable? All other DB Names are hardcoded.

Reply:
Mhm, is there a reason to hardcode it? Some sysadmins like to configure their database names.

Reply:
And this one is a bit special since it is really only for the authorization server. It could really be created in another db server since no other service should have access to it.

Reply:
> Mhm, is there a reason to hardcode it?

Then it would be similar to the other code in SW360.

Reply:
Is there another reason beside that? I don't see right now why hardcoding a db name is better.